src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertCapabilities.sql

```diff
@@ -65,6 +66,11 @@ VALUES
 ('53255CB2-C5F6-427B-A18A-D3055E310FD6', 'C46', 'Appointments Management – COVID-19 Vaccinations', 'The Appointments Management – COVID-19 Vaccinations Capability enables the administration and scheduling of COVID-19 vaccination appointments for Patients.', @covidVaccinationBaseUrl + '7918551324/Appointments+Management+-+COVID-19+Vaccinations', '1.0.0', '2020-12-09', 2),
 ('EC26D316-4F3D-45B1-BDF5-717D24DAB360', 'C47', 'Vaccination and Adverse Reaction Recording', 'The Vaccination and Adverse Reaction Recording Capability enables the recording of COVID-19 vaccination and adverse reaction data at the point of care. The Capability also supports the delivery of this data to the Patient’s registered GP Practice Foundation Solution and to NHS Digital.', @covidVaccinationBaseUrl + '7918551342/Vaccination+and+Adverse+Reaction+Recording', '1.0.0', '2020-12-09', 2);
 
+INSERT INTO @capabilities(Id, CapabilityRef, [Name], [Description], SourceUrl, [Version], EffectiveDate, CategoryId, FrameworkId)
+VALUES
+('EE71409B-F570-4581-B082-2B13DAC6CE6D', 'C43',	'Online Consultation',	'The Online Consultation Capability allows Patients/Service Users/Proxies to request and receive support relating to healthcare concerns, at a time and place convenient for them.', 'https://gpitbjss.atlassian.net/wiki/spaces/DFOCVC/pages/2519925022/Online+Consultation',	'0.5.0', '2021-03-11',	3, 'DFOCVC001'),
+('4E07C901-1DA1-43EB-BE45-C89B7DBD9D66', 'C44',	'Video Consultation',  'The Video Consultation Capability allows Health or Care Professionals to conduct secure live remote video consultations with individual or groups of Patients/Service Users/Proxies ensuring they can receive support relating to healthcare concerns when a Video Consultation is most appropriate.', 'https://gpitbjss.atlassian.net/wiki/spaces/DFOCVC/pages/2519925030/Video+Consultation', '0.5.0', '2021-03-11', 3, 'DFOCVC001');
+
 -- It's unlikely that the code below will need to change unless/until there is more than one framework
```

Comment:
Can you delete this comment. Now that you've updated the `MERGE` statement to work with multiple frameworks I don't think we need it any more.